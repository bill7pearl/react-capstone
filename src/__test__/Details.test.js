import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import GameDetails from '../components/Details';
import store from '../redux/configureStore'

it('matches snapshot', () => {
  const tree = renderer.create(
    <Provider store={store}><GameDetails /></Provider>,
  )
    .toJSON();
  expect(tree).toMatchSnapshot();
});