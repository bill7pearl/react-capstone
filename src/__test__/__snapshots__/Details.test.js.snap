// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<main>
  <ul>
    <li>
      <img />
    </li>
    <div>
      <li>
        Name:
      </li>
      <li>
        Description:
      </li>
      <li>
        Publisher:
      </li>
      <li>
        genre:
      </li>
      <li>
        Release date:
      </li>
    </div>
    <p>
      No Games are Found!
    </p>
  </ul>
</main>
`;
